Configuration/StandardSequences/python/GeometryConf.py

```diff
@@ -69,4 +69,5 @@
     'Extended2026D88' : 'Extended2026D88,Extended2026D88Reco',
     'Extended2026D89' : 'Extended2026D89,Extended2026D89Reco',
     'Extended2026D90' : 'Extended2026D90,Extended2026D90Reco',
+    'Extended2026D91' : 'Extended2026D91,Extended2026D91Reco',
```

Comment:
@srimanob out of curiosity in light of https://github.com/cms-sw/cmssw/pull/36851 should also all new geometries be provided with dd4hep hooks?

Reply:
Thanks @mmusich on this point. I'm not sure. We are trying to focus on ones that uses for relvals at the moment, D77/D86/D88. I think we can leave as it is for a moment until we can make the first complete DD4hep Phase-2 workflows.